src/promptflow/tests/executor/e2etests/test_executor_happypath.py

```diff
@@ -106,6 +107,19 @@ def test_executor_exec_node(self, flow_folder, node_name, flow_inputs, dependenc
         assert run_info.node == node_name
         assert run_info.system_metrics["duration"] >= 0
 
+    def test_executor_exec_node_with_llm_node(self, dev_connections):
+        # Run the test in a new process to ensure the openai api is injected correctly for the single node run
+        queue = multiprocessing.Queue()
+        process = multiprocessing.Process(
```

Comment:
Here we need to make sure the start method of the process is spawn. Fork will also inherit the current environment.

Reply:
Updated.